source/slang/slang-compiler.h

```diff
@@ -2733,11 +2733,14 @@ namespace Slang
 
         SlangResult emitPrecompiledDXIL(ComPtr<IArtifact>& outArtifact);
 
+        SlangResult emitPrecompiledSPIRV(ComPtr<IArtifact>& outArtifact);
```

Comment:
This is fine for now, but we may want to consider unifying the dxil/spirv methods with a single `emitPrecompiledDownstreamIR(CodeGenTarget target, ComPtr<IArtifact>& outArtifact);`.

Reply:
Unified